data/campaigns/Eastern_Invasion/scenarios/08_Xenophobia.cfg

```diff
@@ -0,0 +1,837 @@
+                    scroll=no
+                [/message]
+            [/else]
+        [/if]
+        [message]
+            speaker=Prok-Bak
+            message= _ "Don’t bother asking those freeloading dwarves for anything! They can’t even manage to pay their tribute on time."
+        [/message]
+        [message]
+            speaker=Pelathsil
+            #po: "To flames with your tribute! What's ours is ours, and don't any of you orcs or humans try to take it."
+            message= _ "To flames with yer tribute! What’s ours be ours, and don’t no orcs nor any o’ ye humans be tryin and takin’ it!"
+        [/message]
+        [if]
+            [have_unit]
+                id=Dolburras
+            [/have_unit]
+
+            [then]
+                [message]
+                    speaker=Dolburras
+                    #po: "No, these humans are good folk! You can trust us to help against the orcs. All we ask is some food and drink to reprovision."
+                    message= _ "Nay, these humans be good folk! Ye can trust us to help against them orcs. All we be askin’ be some food and drink to reprovision."
+                [/message]
+            [/then]
+
+            [else]
+                [message]
+                    speaker=Gweddry
+                    message= _ "Can we not work together, for our mutual benefit? All we ask is some food and drink to reprovision."
+                [/message]
+            [/else]
+        [/if]
+        [message]
+            speaker=Pelathsil
+            #po: "Never! What's ours is ours! Not since the days of Ulrek and Delfador have any of our clan helped one of you gangly humans!"
+            message= _ "Never! What’s ours be ours! Not since tha days o’ Ulrek and Delfador ha’ any o’ our clan helped one o’ ye gangly humans!"
+        [/message]
+        [message]
+            speaker=Aleii
+            message= _ "Hsss... orcsss, dwarvesss, humansss. We have traveled far to find thisss warmth. I have called the vote; the clutch has ssspoken. These waters are only for Naga. Leave usss be."
+        [/message]
+        [message]
+            speaker=Prok-Bak
+            message= _ "That’s Whitefang territory you’re squatting in, snake! Show some respect for the Chief!" # whitefangs from DiD, in roughly the same area
+        [/message]
+        [message]
+            speaker=Bagork
+            message= _ "And I bet they haven’t paid tribute either! Dwarves, snakes, humans, I say we kill ‘em all!"
+        [/message]
+        [if]
+            [have_unit]
+                id=Hahid al-Ali
+            [/have_unit]
+
+            [then]
+                [message]
+                    speaker=Hahid al-Ali
+                    message= _ "Barbarians, the lot of you! I’m going to starve because you’re busy fighting like a hatch of desert scorplings. <span size='small'>Don’t know what I expected, throwing in my lot with a bunch of primitive northerners...</span>"
```

Comment:
Should 'northerners' be capitalised?

Reply:
@stevecotton Reason for disagreeing?

Reply:
At the moment, races aren't capitalised, nor are insults such as "tree shaggers" or "fish".

Proper nouns such as Northern Elves are capitalised, but here "northerners" seems to be a name that dunefolk use for non-dunefolk humans, rather than a proper noun.

Reply:
I was considering that 'northerners' are a group that would be treated as a proper noun, just as with Northern Alliance and Northern Elves. Even from the perspective of the dunefolk. There's even a music track titled Northerners, if I recall correctly (though not particularly relevant).

Anyway, it was just a question. If we deem it not a proper noun, then close all my other remarks on the same word (there are a lot of them).

I wouldn't have considered insults to be capitalised.

Reply:
It's a difficult word, because here "northerners" is specifically human loyalists, whereas in other contexts it would refer to dwarves. Might even be inconsistent with some other campaign having a group using it as a proper noun.